libtiledbvcf/src/c_api/tiledbvcf.cc

```diff
@@ -32,13 +32,16 @@
 
 #include "c_api/tiledbvcf.h"
 #include "read/reader.h"
+#include "utils/logger_public.h"
 #include "utils/utils.h"
 #include "write/writer.h"
 
 #include <cassert>
 #include <iostream>
 #include <memory>
 
+using namespace tiledb::common;
```

Comment:
Why are we using this namespace here? This is an internal namespace to TileDB

Reply:
That namespace came with the `logger.*` files from the core. I'll change the namespace in this repo we don't use the TileDB internal namespace.